Files/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -484,7 +484,6 @@
                     AllowDrop="{x:Bind InstanceViewModel.IsPageTypeSearchResults, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}"
                     CanDragItems="{x:Bind InstanceViewModel.IsPageTypeSearchResults, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}"
                     ContainerContentChanging="FileList_ContainerContentChanging"
-                    DesiredWidth="{x:Bind GridViewItemMinWidth, Mode=OneWay}"
```

Comment:
This change disables stretching items so that they fit the width of the layout (#4940). Is this intentional?

![image](https://user-images.githubusercontent.com/59544401/128454931-37508560-a8ba-4c14-99d2-936916508b8c.png)

Reply:
Sorry, I didn't notice that feature.